mypy/semanal.py

```diff
@@ -3251,12 +3251,15 @@ def name_already_defined(self, name: str, ctx: Context) -> None:
         self.fail("Name '{}' already defined".format(name), ctx)
 
     def fail(self, msg: str, ctx: Context, serious: bool = False, *,
-             blocker: bool = False) -> None:
+             blocker: bool = False, implicit_any: bool = False) -> None:
```

Comment:
I agree. Unfortunately, I need to suppress the error message based on the information (`options`, and whether the module being checked is a stub) that's unavailable inside `TypeAnalyser` methods where the error occurs.  I am not sure what the best fix for this would be. If I don't add an extra parameter to `fail`, I'll have to pass all this information into `TypeAnalyser`, but that's not good either since `TypeAnalyser` should not care about those details.